src/navigator-card-body.jsx

```diff
@@ -280,7 +287,24 @@ export const NavigatorCardBody = ({
                 <Spinner />
             </Flex>
         );
-    if (isGrid) {
+    const clearFilter = () => {
+        setCurrentFilter("");
+    };
+    if (sortedFiles.length === 0) {
```

Comment:
I feel like this is probably also going to show on empty folders, even if there's no search term?

Probably we should restrict this to showing with non-empty search.

Reply:
You're right. I made a new commit. Could you please look into it?